YSI_Visual/y_interiors/y_interiors_impl.inc

```diff
@@ -0,0 +1,300 @@
+#include "y_interiors_decl"// decl-arations
+
+/*
+
+                                                              
+88        88                            88                    
+88        88                            88                    
+88        88                            88                    
+88aaaaaaaa88   ,adPPYba,    ,adPPYba,   88   ,d8   ,adPPYba,  
+88""""""""88  a8"     "8a  a8"     "8a  88 ,a8"    I8[    ""  
+88        88  8b       d8  8b       d8  8888[       `"Y8ba,   
+88        88  "8a,   ,a8"  "8a,   ,a8"  88`"Yba,   aa    ]8I  
+88        88   `"YbbdP"'    `"YbbdP"'   88   `Y8a  `"YbbdP"'  
+                                                              
+                                                              
+
+*/
+
+#define YSI_INTERIORS_TRUE 1

```

Comment:
Why are these defines needed?  Could the API not be simplified to just accept `true`/`false`?